site/content/extend/plugins/best-practices.md

```diff
@@ -6,11 +6,15 @@ weight: 50
 
 See here for [server-specific best practices for plugins](/extend/plugins/server/best-practices/). Webapp-specific best practices are incoming.
 
-## How can I create my own plugin setting?
+## How do I enable plugin configuration through the System Console?
 
-A plugin's admin settings are defined in the plugin's manifest [setting schema](https://developers.mattermost.com/extend/plugins/manifest-reference/#settings_schema). This includes any pre-defined types that the System Console UI currently supports. 
+Once a plugin is installed, Administrators have access to the plugin's configuration page in the __System Console > Plugins__ section. The configurable settings must first be defined in the plugin's manifest [setting schema](https://developers.mattermost.com/extend/plugins/manifest-reference/#settings_schema). The web app supports several basic pre-defined settings type, e.g. `bool` and `dropdown`, for which the corresponding UI components are provided in order to complete configuration in the System Console.
 
-Additionally, a plugin could define their own type of setting with a corresponding custom user interface that will be displayed in the system console following the process below. 
+These settings are stored within the server configuration under [`Plugins`] indexed by plugin ids. The plugin's server code can access their current configuration calling the [`getConfig`](https://developers.mattermost.com/extend/plugins/server/reference/#API.GetConfig) API call and can also make changes as needed with [`saveConfig`](https://developers.mattermost.com/extend/plugins/server/reference/#API.SaveConfig).
+
+## How can I define my own plugin setting type?
+
+A plugin could define its own type of setting with a corresponding custom user interface that will be displayed in the System Console following the process below.  
```

Comment:
nit: both of these section headers have "I" in the title, but then we don't go on to say "you" anywhere. In particular, I stumbled over the subject change here, almost equating "I" with "plugin".

Reply:
True, maybe I was just following the "theme" of the original question that was in this section [How can I review the entire code base of a plugin?](https://developers.mattermost.com/extend/plugins/best-practices/#how-can-i-review-the-entire-code-base-of-a-plugin)

Do you think it's more proper to rephrase to maybe something like
```suggestion
How can a plugin define its own setting type?
```

Reply:
I'd be fine with that phrasing :)

Reply:
Done!